### DISCUSSION TITLE
Trying to use GRDB for the first time to replace Realm from my app - should I directly go with GRDB 7?

### DISCUSSION THREAD

**Discussion by hyouuu** [groue/GRDB.swift#1659] [Q&A] [answered]
Looks like it's already mature are we releasing it soon?

**Comment by groue** [comment, marked as answer]
Hello @hyouuu.

You can go ahead with the beta indeed. There's isn't much risk of bug, or of breaking changes :-)

```swift
dependencies: [
    .package(url: "https://github.com/groue/GRDB.swift", from: "7.0.0-beta"),
]
```

**Reply by hyouuu** [reply]
Awesome thanks for the quick confirmation!

**Reply by freak4pc** [reply]
Following up with this question - indeed no issues of breakage or bugs, but any ETA for an RC release?

**Reply by groue** [reply]
Hi @freak4pc! I can't give any ETA but I can tell what's holding the release. I have been very busy in the last few months but things are clearing up now. There's an issue or a discussion about JSONB, and I just want to spend some time on it in case I would have to introduce a breaking change in the serialization/deserialization layer. And finally there's this `Sendable` requirement in values consumed and returned by async database access methods that is really bugging me. It's really hard to understand what's needed to relax it (maybe some breaking changes, unfortunately).

**Reply by groue** [reply]
> And finally there's this `Sendable` requirement in values consumed and returned by async database access methods that is really bugging me.

I won't address it. It's intractable and I'll need help from God knows who.

I'll look at JSONB shortly.

**Reply by groue** [reply]
GRDB 7.0.0 is out 🚀
